tests/xlsx/xlsx_import_export.test.ts

```diff
@@ -311,6 +311,7 @@ describe("Export data to xlsx then import it", () => {
       legendPosition: "top" as const,
       stacked: false,
       labelsAsText: false,
+      showValues: true,
```

Comment:
Maybe we could add some chart without showValues ? So that we have `showValues: false` at some places here ?